recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+
+build:
+  number: 0
+  skip: True        # [win]
```

Comment:
Seems the CIs are running out of time with the different `python` and `numpy` requirements. Would recommend skipping NumPy 1.11 for now by changing `win` -> `win or np!=112`. Can remove this in the feedstock and re-render to get the older NumPy version into the build matrix.